fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -1149,6 +1149,17 @@ public class Config extends ConfigBase {
     @ConfField(mutable = true, aliases = {"disable_colocate_balance"})
     public static boolean tablet_sched_disable_colocate_balance = false;
 
+    /**
+     * When setting to true, disable the overall balance behavior for colocate groups which treats all the groups
+     * in all databases as a whole and balances the replica distribution between all of them.
+     * See `ColocateBalancer.relocateAndBalanceAllGroups` for more details.
+     * Notice: set `tablet_sched_disable_colocate_balance` to true will disable all the colocate balance behavior,
+     * including this behavior and the per-group balance behavior. This configuration is only to disable the overall
+     * balance behavior.
+     */
+    @ConfField(mutable = true)
+    public static boolean tablet_sched_disable_colocate_overall_balance = false;
```

Comment:
```suggestion
    public static boolean tablet_sched_enable_colocate_overall_balance = true;
```
use the proactive syntax.

Reply:
i have thought about this, since we already have a configuration named `tablet_sched_disable_colocate_balance` and this new configuration has similiar meaning with it, i think it's more important to keep in consistent with the convention first.

Reply:
It's better to gradually use enable to replace disable. The `enable` is more convenient to understand.

Reply:
i agree, my point is that, if it's a pure new configuration, i will use the proactive syntax without doubt, but it has strong relationship wih `tablet_sched_disable_colocate_balance`, which is reactive syntax, so here keeping consistency is more important.